### DISCUSSION TITLE
Correct procedure to import model exported from another map file -TinyButton

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#197] [Q&A] [answered]
I want to add a TinyButton entity to CorporatePenthouse map (Mission 17) where it seems this does not exist in the file.
I can reproduce the composite data from another map's file,  but I see there's a model reference and went down a rabbit hole;
I exported the TinyButton.cs2 from another map in this case Tech_Hub.    It exports as a file tinybutton_display.fbx.

When I try to import it to corporatepenthouse I get an error that it "failed to generage CS2 from selected mode. 
Could not find mesh data! please ensure all meshes are childred of the scene's root node."   
Screen shot attached.   
![image](https://github.com/MattFiler/OpenCAGE/assets/51859157/4763c466-c826-4ae2-924c-5606c8b82b2f)

So am I doing something wrong here or is this a bug ?  I don't want to make an uncessary ticket.

I thought you wrote some detailed instructions on adding assets but I can't find them.

**Comment by MattFiler** [comment, marked as answer]
Hey! 

This is due to the current requirements for model data to be contained within submeshes of the root node within the model hierarchy. When you export a model using the tools it adds an additional node layer into the model file.

I admit this is kinda dumb and I'll definitely be changing it in the next update to make importing/exporting models within AI itself easier in future - I'll open an enhancement ticket to cover this.

**Comment by ghramsey** [comment]
Thank you much.  Hopefully not too much extra work.

I have more to add on this, but I'll keep it on the #198 ticket since in involves that bug.
